supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesWorkflowPaths.scala

```diff
@@ -13,22 +17,36 @@ object JesWorkflowPaths {
   def apply(workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration, gcsFileSystem: GcsFileSystem) = {
     new JesWorkflowPaths(workflowDescriptor, jesConfiguration, gcsFileSystem)
   }
-}
 
-class JesWorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration, gcsFileSystem: GcsFileSystem) {
+  implicit class GoogleAuthWorkflowOptions(val workflowOptions: WorkflowOptions) extends AnyVal {
```

Comment:
Can't this use the copy of this implicit class in the package object?  We have three copies of this already... 😢